skyportal/services/test_server/test_api_server.py

```diff
@@ -14,6 +15,22 @@
 from baselayer.log import make_log
 
 
+def get_cache_file_static():
+    """
+    Helper function to get the path to the VCR cache file for requests
+    that must be updated by hand in cases where regular refreshing is
+    infeasible, i.e. limited access to the real server.
+
+    To update this server recording:
+    1) delete the existing recording
+    2) re-run all tests (with API keys for telescopes in place)
+    3) replace any secret information (such as API keys) with dummy values
```

Comment:
I guess this step could be automated from the configuration?

Reply:
Maybe @kmshin1397 also has thoughts on this one. I think the main issue is that the programs will change with time, so care must be taken no matter what.

Reply:
Yes, this is already automated through the configurations by default. It's just hard to do it in this case since as Michael says the programs will change and we are relying on real time assigned on real accounts instead of test allocations to record these responses.